start from scratch backend/src/middleware/authMiddleware.js

```
@@ -0,0 +1,39 @@
+const jwt = require('jsonwebtoken');
+const User = require('../models/User');
+
+// Protect routes
+exports.protect = async (req, res, next) => {
+    let token;
+
+    if (
+        req.headers.authorization &&
+        req.headers.authorization.startsWith('Bearer')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)